src/construction.cpp

```diff
@@ -564,7 +563,7 @@ void construction_menu()
             }
             if( chosen < ( int )constructs.size() ) {
                 if( player_can_build( g->u, total_inv, constructs[chosen] ) ) {
-                    struction( constructs[chosen] );
+                    place_construction( constructs[chosen] );
```

Comment:
Well, I accidentally did search and replace for place_construction and forgot to undo just this one result.